src/query/query_parser/query_parser.rs

```diff
@@ -182,7 +189,8 @@ pub struct QueryParser {
     default_fields: Vec<Field>,
     conjunction_by_default: bool,
     tokenizer_manager: TokenizerManager,
-    boost: HashMap<Field, Score>,
+    boost: FxHashMap<Field, Score>,
+    fuzzy: FxHashMap<Field, (bool, u8, bool)>,
```

Comment:
Can you create a struct for the (bool, u8, bool) tuple? With two bools it's easy to confuse them

Reply:
Done.